src/lib/journey/_utilities/step.utilities.ts

```diff
@@ -0,0 +1,70 @@
+import {
```

Comment:
A lot of this file contains functions that have been moved out of the `journey.store.ts` file, so the logic can be tested independently. Much of this is for determining when and how to handle the varying levels of errors in a journey. Once such error is the DS error when IDM fails to validate the callbacks.